Libraries/acdriver/Headers/acdriver/Driver.h

```diff
@@ -14,6 +14,7 @@
 #include <vector>
```

Comment:
I want to avoid it in pbxbuild, so it's more usable outside a single process. For example if you built an IDE that used pbxbuild for building, it shouldn't need to spawn another process to run the build.